### PR TITLE
I added support for rotated tiles in Tiled Component

### PR DESCRIPTION
**Pull request by Arne1303**
I have added some suggestions after the first PR.
This Change needs a PR I issued to flame-engine/tiled.dart to work

### CODE REVIEW

**Comment by erickzanardo** [comment]
You don't need to open a new PR to add the changes, you could just submit to the old branch.

Also, please, link the PR on tiled.dart here.

**Comment by Arne1303** [comment]
Sorry, I didn't know that. 

Here is the other PR
https://github.com/flame-engine/tiled.dart/pull/5

File: lib/components/tiled_component.dart [hunk 1]
```diff
@@ -58,6 +59,38 @@ class TiledComponent extends Component {
...
+    map.objectGroups.forEach((objects) {
+      if (objects.visible) {
+        _renderObjectGroup(c, objects);
+      }
+    });
+  }
+
+  void _drawRotatedImageRect(Canvas canvas, src, dst, image, Offset origin, {double rotate = 0, double scaleX = 1, double scaleY = 1}) {
+    canvas.save();
+    canvas.translate( origin.dx, origin.dy );
```
Comment (luanpotter) [inline lib/components/tiled_component.dart:72]: please remove spaces right after and before the parenthesis, here and elsewhere
```
foo(bar, 1, false)
```

File: lib/components/tiled_component.dart [hunk 2]
```diff
@@ -58,6 +59,38 @@ class TiledComponent extends Component {
...
+      if (tmx.gid != null && tmx.visible) {
+        final tile = map.getTileByGID(tmx.gid);
+        final image = images[tile.image.source];
+
+        final rect = tile.computeDrawRect();
+        final src = Rect.fromLTWH(rect.left.toDouble(), rect.top.toDouble(),
+            rect.width.toDouble(), rect.height.toDouble());
+        final dst = Rect.fromLTWH(tmx.x.toDouble(), tmx.y.toDouble()-rect.height.toDouble(),
+            rect.width.toDouble(), rect.height.toDouble());
+        _drawRotatedImageRect(c, src, dst, image, dst.bottomLeft,rotate: tmx.rotation.toDouble()*(math.pi/180));
```
Comment (luanpotter) [inline lib/components/tiled_component.dart:91]: please apply dart format to this file, there some inconsistencies (eg operator spacing)

**Comment by luanpotter** [comment]
Closing in favour of #403

**Pull request closed without merging**